src/WebGPU/Silk.NET.WebGPU/Platforms/MacOS/CAMetalLayer.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Silk.NET.WebGPU.Platforms.MacOS;
+
+internal struct CAMetalLayer
+{
+    public readonly IntPtr NativePtr;
```

Comment:
Silk.NET code tends to prefer `nint` or `void*` over `IntPtr`

Reply:
Switched to nint